README.md

```diff
@@ -50,6 +50,8 @@ layer-profile:
       dataset: [the name of your dbt dataset]
       threads: [1 or more]
       keyfile: [/path/to/bigquery/keyfile.json]
+      layer_project: [the Layer project to use for model training (opt)]
```

Comment:
Can we use the dbt project name? I did it on automl see here:
https://github.com/layerai/dbt-adapters/blob/090a5748a997ab135754d4d12e7df662c3c9dab1/common/adapter.py#L184

Reply:
We could use the dbt project name as default if `layer_project` is not provided.

Reply:
Sure, it makes sense!

Reply:
Done.